cmd/capture.go

```diff
@@ -70,9 +70,15 @@ func runCaptureOfflineMode(opt *captureOpt) error {
 			return fmt.Errorf("connect to DB1 error: %v", err)
 		}
 	} else {
-		db1, err = startAndConnectDB(opt.db1, opt.DB)
+		db1, err = startDB(opt.db1)
 		if err != nil {
-			return fmt.Errorf("start and connect to DB1 error: %v", err)
+			return fmt.Errorf("start DB1 error: %v", err)
+		}
+		if err := importSchemaStats(db1, "", opt.schemaDir); err != nil {
```

Comment:
There is a confusing point, why is the schema and stats not imported for db1 in offline mode before? Is there any special consideration?

Reply:
Not very sure now... it seems that we just forget to import stats before...